.github/stale.yml

```diff
@@ -0,0 +1,58 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+# daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "kind/feature"
```

Comment:
so you'd never want feature requests go stale?

Reply:
Dunno, I put it here to start discussion :smile:

Reply:
was just confirming.
Looks like a good way to start.
+1

Reply:
I think this makes sense, also there are barely any issues open with feature label, 12 currently so those are easy to go through manually.